packages/legacy/core/App/navigators/defaultStackOptions.tsx

```diff
@@ -8,6 +8,11 @@ export function createDefaultStackOptions({ ColorPallet }: ITheme) {
     headerTintColor: ColorPallet.brand.headerIcon,
     headerShown: true,
     headerBackTitleVisible: false,
+    headerTitleContainerStyle: {
+      maxWidth: '72%',
+      minWidth: '72%',
+      width: '100%',
```

Comment:
What is the result of setting all three of these?

Reply:
Good question. I may have left that when experimenting. I won't change it for this PR since it works fine and I'd have to retest on a much of screens. But, for my current feature I'll confirm what is the minimum set required.